src/edu/csus/ecs/pc2/core/execute/Executable.java

```diff
@@ -2416,6 +2538,8 @@ public String replaceString(String origString, String beforeString, int afterInt
      *              {:outfile}
      *              {:ansfile}
      *              {:pc2home}
+     *              {:sandbox} -the sandbox program (only under non-Windows OS's)
```

Comment:
doc, Unclear why there is a windows only variable?
I suggest removing this reference to {:sandbox}

Reply:
Well, it's not that it's "Windows-only", it's that it's "Not Windows".  Anyway, I wrote this comment to make it clear that the "{:sandbox}" substitution in only used on non-Windows systems.  But I grant it's a bit confusing; I've eliminated the reference.  I also corrected the substitution variable names.